go/store/datas/pull.go

```diff
@@ -40,21 +40,24 @@ import (
 
```

Comment:
Lets kill all the changes in this file as they are no longer applicable in the latest and will just cause conflicts.